packages/react-native-usb/android/src/main/java/io/trezor/rnusb/ReactNativeUsbModule.kt

```diff
@@ -150,7 +150,8 @@ class ReactNativeUsbModule : Module() {
             for (device in devicesList) {
                 if (usbManager.hasPermission(device)) {
                     Log.d(LOG_TAG, "Has permission, send event onDeviceConnected: $device")
-                    val webUsbDevice = openDevice(device.deviceName)
+                    openDevice(device.deviceName)
+                    val webUsbDevice = getWebUSBDevice(device)
```

Comment:
coudn't these two lines be replaced with `getOpenedConnection` ?

Reply:
We need `WebUSBDevice` here, and `getOpenedConnection` returns `UsbDeviceConnection`, so I don't see it as a replacement.